src/main/java/org/betonquest/betonquest/quest/event/give/GiveEvent.java

```diff
@@ -64,23 +64,28 @@ public GiveEvent(final Item[] questItems, final NotificationSender itemsGivenSen
     @Override
     public void execute(final Profile profile) throws QuestRuntimeException {
         final Player player = profile.getOnlineProfile().get().getPlayer();
-        Arrays.stream(questItems).toList().forEach(item -> {
+        for (final Item item : Arrays.stream(questItems).toList()) {
```

Comment:
Why converting the array to a list before iterating over it? I see no point.  
This just makes it confusing to read